UnityProject/Assets/Scripts/Managers/AddressableManager/AddressableCatalogueManager.cs

```diff
@@ -246,9 +246,9 @@ public static List<string> GetCataloguePathStreamingAssets()
 		var directories = System.IO.Directory.GetDirectories(pathss);
 		var catalogues = new List<string>();
 		var multiCatalogues = new List<string>();
-		foreach (var Directorie in directories)
+		foreach (var directorie in directories)
```

Comment:
![image](https://user-images.githubusercontent.com/43683714/165883942-177ab5fd-a7de-41b1-8e38-f5f76beb4ff6.png)
when I google this version of the word I don't get any result